data-tiers.md

```diff
@@ -0,0 +1,59 @@
+# Data Tier Definitions
+
+This document defines the data tiers within Dogpark from the perspective of Retriever requirements and interface guarantees.
+It will not prescribe hosting technologies.
+
+## Tier 0
+
+Tier 0 shall operate under the following requirements:
+
+Modelling/content:
+
+- Pre-conformed to BioLink model and node-normalized
+- Knowledge graph is not partitioned by source
+
+Usage:
+
+- Higher performance than Tier 1
+- Sub-queried by Retriever in all normal cases of Shepherd queries
+- Supports multi-hop graph queries
+- Interfaces with Retriever through a query language determined by its hosting technology, not TRAPI
+
+## Tier 1
+
+Tier 1 shall operate under the following requirements:
+
+Modelling/content:
+
+- Pre-conformed to BioLink model and node-normalized
+- Not necessarily a single unified knowledge graph
+- Update pipeline is significantly faster than Tier 0
```

Comment:
As a clarification, how about "KGs in Tier 1 are independently managed and served, potentially resulting in a more frequent additions/updates."

Reply:
See update [here](https://github.com/NCATSTranslator/Data-Access-Working-Group/pull/6/commits/72f5a9700113255de5fcc3b40d3eb77211276d92#diff-b6ac0e4a997fc9683fbf1e2dfb36ad9056fcec23c1ea65bfc3b864b0fe19bdeaL29-R45)